app/src/main/java/com/mirandadayadkins/junglee/FriendsListActivity.kt

```diff
@@ -0,0 +1,52 @@
+package com.mirandadayadkins.junglee
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+import com.mirandadayadkins.junglee.databinding.ActivityFriendsListBinding
+
+class FriendsListActivity : AppCompatActivity() {
+
+    // Set up ViewBinding for the layout
+    private lateinit var binding: ActivityFriendsListBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        // Setting up binding instance and view instances
+        binding = ActivityFriendsListBinding.inflate(layoutInflater)
+        val view = binding.root
+        // Navigation bar view instances
+        val myJungleNavBarButton = binding.myJungleButton
+        val landingPageNavBarButton = binding.homeButton
+        val environmentNavBarButton = binding.environmentButton
+        val checklistNavBarButton = binding.checklistButton
+
+        // Setting onClickListeners for Navigation Bar Buttons
+
+        myJungleNavBarButton.setOnClickListener {
```

Comment:
Repeating these everywhere is a violation of the dry principle

Reply:
fixed